### PR TITLE
[OSDOCS-5469]: Changing title of "Identity and access management."

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OSDOCS-5469
Link to docs preview:

https://57465--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html#rosa-policy-responsibilities_rosa-policy-responsibility-matrix

https://57465--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html#rosa-policy-identity-access-management_rosa-policy-responsibility-matrix
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
\label qe-review-needed

**Comment by yuwang-RH** [comment]
LGTM～

**Review by arendej — APPROVED** [review]
Thanks. lgtm

File: modules/policy-shared-responsibility.adoc [hunk 1]
```diff
@@ -96,8 +96,8 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
 |===
 
 [id="identity-access-management_{context}"]
-== Identity and access management
-The Identity and Access Management matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
+== Access and identity authorization
+The Access and Identity Authorization matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
```
Comment (eohartman) [inline modules/policy-shared-responsibility.adoc:100]: The capitalization of "Access and Identity Authorization" should be set to lowercase.

File: modules/rosa-policy-shared-responsibility.adoc [hunk 2]
```diff
@@ -96,8 +96,8 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
 |===
 
 [id="rosa-policy-identity-access-management_{context}"]
-== Identity and access management
-The Identity and Access Management matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
+== Access and identity authorization
+The Access and Identity Authorization matrix includes responsibilities for managing authorized access to clusters, applications, and infrastructure resources. This includes tasks such as providing access control mechanisms, authentication, authorization, and managing access to resources.
```
Comment (eohartman) [inline modules/rosa-policy-shared-responsibility.adoc:100]: The capitalization of "Access and Identity Authorization" should be set to lowercase.

**Review by eohartman — CHANGES_REQUESTED** [review]

File: modules/policy-shared-responsibility.adoc (hunk 1 above)
Comment (mletalie) [reply modules/policy-shared-responsibility.adoc:100]: Hello @eohartman,
    Thanks for the input. I was wondering if the original text (with caps) was incorrect, but decided to make my changes and go with the original formatting. I will make the requested changes asap and await our review. Thanks!

**Comment by mletalie** [comment]
Hello @eohartman,
    Suggested changes have been made and committed. Thanks!

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM as well. Merging!

**Pull request merged**